ocamltest/main.ml

```diff
@@ -121,13 +121,9 @@ let get_test_build_directory_prefix test_dirname =
   if test_dirname = "." then root
   else Filename.concat root test_dirname
 
-let tests_to_skip = ref []
-
-let init_tests_to_skip () =
-  tests_to_skip := String.words (Sys.safe_getenv "OCAMLTEST_SKIP_TESTS")
-
-let test_file test_filename =
-  let skip_test = List.mem test_filename !tests_to_skip in
+let tests_to_skip = String.words (Sys.safe_getenv "OCAMLTEST_SKIP_TESTS") in
```

Comment:
```suggestion
let tests_to_skip = String.words (Sys.safe_getenv "OCAMLTEST_SKIP_TESTS")
```
I don't know what @shindere's opinion is, but it looks like the purpose of what was there before was to have initialisation code later on. It can't fail, but there's no reason not to have `tests_to_skip` defined in or just before the `let ()` at the bottom of this file.

Reply:
Yes, I am making the changes currently and once it compiles successfully, I will push. Thanks for your suggestion